electrum/payment_identifier.py

```diff
@@ -463,18 +468,35 @@ async def _do_notify_merchant(
             if on_finished:
                 on_finished(self)
 
-    def get_onchain_outputs(self, amount):
+    def get_onchain_outputs(self, amount, bip21_use_fallback=False):
```

Comment:
Instead of `bip21_use_fallback` I'd prefer a more explicit `allow_silent_payment' kwarg.